pkg/printers/humanreadable.go

```diff
@@ -399,7 +406,14 @@ func PrintTable(table *metav1beta1.Table, output io.Writer, options PrintOptions
 // does not expose metadata). It returns an error if the table cannot
 // be decorated.
 func DecorateTable(table *metav1beta1.Table, options PrintOptions) error {
+	return DecorateTableForEvent("", table, options)
+}
+
+func DecorateTableForEvent(eventType string, table *metav1beta1.Table, options PrintOptions) error {
```

Comment:
Kind of yucky.  A table should know its column structure, and an event should be `<underlying printer> + event type wrapper`, so I feel like this shouldn't be necessary.

Reply:
Move the event printing comments to https://github.com/kubernetes/kubernetes/pull/72416

Will rebase this on master so it's just the tabwriter change